src/ConfigEditor/AuthTypeSwitch.tsx

```diff
@@ -38,8 +37,8 @@ export function AuthTypeSwitch({ useManagedSecret, onChangeAuthType }: Props) {
     >
       <RadioButtonGroup
         options={[
-          { label: 'Temporary credentials', value: false },
```

Comment:
TBH I won't recommend temporary credentials for Redshift Serverless because we can't use redshift_data_api_user. 

It's because of the design level difference of temporary credential auth for Serverless and Provisioned as described [here](https://docs.aws.amazon.com/redshift/latest/mgmt/data-api-access.html):

- Provisioned take database user as a input: `redshift_data_api_user`
- Serverless does not take database user as an input. It maps IAM user or a role to a database user: `IAM:sample_user` or `IAMR:sample_role`. 
  - Therefore customers will need to check the mapped username and GRANT read-only superuser privilege to the database user (which is not so convenient)...

Reply:
Hmm interesting, I guess if we wouldn't recommend using serverless with temporary credentials then perhaps we shouldn't make that an option? 

If we want to promote the use of SecretsManager in general for serverless or nonserverless accounts, that feels like something we could consider in another pr maybe?

Reply:
Okay let me revert this change. Maybe I'll open an another PR.

Reply:
Sure I've reverted this change at a4f8a1b89a3ecbba33638be43fc8ec128ddcda54. Maybe I'll open an another PR.